src/VisualStudio/Core/Def/TableDataSource/VisualStudioBaseDiagnosticListTable.LiveTableDataSource.cs

```diff
@@ -208,12 +199,8 @@ private void OnDiagnosticsUpdated(object sender, ImmutableArray<DiagnosticsUpdat
                         continue;
                     }
 
-                    // if we're in lsp mode we never respond to any diagnostics we hear about, lsp client is fully
-                    // responsible for populating the error list.
-                    var diagnostics = GlobalOptions.IsLspPullDiagnostics()
-                        ? ImmutableArray<DiagnosticData>.Empty
-                        : e.Diagnostics;
-
+                    // We no longer have solution-crawler mode.  So we don't do anything here.
+                    var diagnostics = ImmutableArray<DiagnosticData>.Empty;
                     if (diagnostics.Length == 0)
```

Comment:
💡 This can be further inlined

Reply:
this file is entirely removed in the followup PR.  so i'll leave till then :)